actions/docker-build-push/action.yaml

```diff
@@ -0,0 +1,201 @@
+name: Docker Build and Push
+description: Build and optionally push a Docker image to a container registry (ghcr, ECR)
+
+inputs:
+  dockerfile_path:
+    required: false
+    default: "./Dockerfile"
+    description: "Dockerfile path"
+  dockerfile_context:
+    required: false
+    default: "."
+    description: "Dockerfile build context"
+  docker_image_name:
+    required: false
+    default: ${{ github.repository }}
+    description: "Docker image name, defaults to repository name"
+  docker_image_description:
+    required: true
+    description: "Image description set as image label"
+  docker_image_authors:
+    required: false
+    default: ""
+    description: "Authors names set as image label"
```

Comment:
question: is this a comma separated list? I'd state it in the description in case